redisinsight/api/src/utils/feature-version-filter.helper.spec.ts

```diff
@@ -0,0 +1,65 @@
+// Import the function to test
+import { FeatureConfigFilterCondition } from 'src/modules/feature/model/features-config';
+import { filterVersion } from './feature-version-filter.helper';
+
+describe('filterVersion', () => {
```

Comment:
❤️ 

Since these tests work with the `path` version (`major`.`minor`.`patch`) - we automatically have tests for the `minor` as well 😸